buildme

```diff
@@ -1,19 +1,28 @@
 #!/bin/bash
 BUILDTYPE=Release
+ARCH=`arch`
+ARM64=OFF
+CMAKE_TOOLCHAIN_FILE=../../../makefiles/cmake/toolchains/arm-linux-gnueabihf.cmake
 
 if [ "$1" = "--debug" ]; then
 	BUILDTYPE=Debug
 	shift
 fi
 
+if [ "$1" = "--aarch64" ]; then
+	ARM64=ON
+	CMAKE_TOOLCHAIN_FILE=../../../makefiles/cmake/toolchains/aarch64-linux-gnu.cmake
+	shift
+fi
+
```

Comment:
No ability to have a debug 64 bit build?

Reply:
Yes you can. ```./buildme --debug --aarch64```.
```shift``` removes the first command line argument and shifts the others up.

It is true that ```./buildme --aarch64 --debug``` will give you a 64-bit release build due to the parsing order. I'm not a bash expert so not intending to fix that one.

Reply:
Agreed - that a debug build is possible without editing the script is enough.

Reply:
I had completley missed the shift, sorry.

Reply:
Shift happens.